common/supplemental/supplementalData.xml

```diff
@@ -4304,8 +4304,8 @@ XXX Code for transations where no currency is involved
         <calendar type="gregorian">
            <calendarSystem type="solar" />
            <eras>
-               <era type="0" end="0-12-31" code="gregory-inverse" aliases="bc bce"/>
-               <era type="1" start="1-01-01" code="gregory" aliases="ad ce"/>
+               <era type="0" end="0-12-31" code="bce" aliases="bc"/> <!-- Before Common Era, Before Christ -->
```

Comment:
@sffc why is `type: 0` used for the backwards-counting, earlier era here, but is used for the forwards-counting, later era in most (all?) calendars below?  Are 0 vs. 1 meaningful in any way?

Reply:
They are generally chronological with the exception of AH/BH, which had a whole discussion in https://github.com/unicode-org/cldr/pull/4581